src/jsrm/systems/utils.py

```diff
@@ -124,3 +124,86 @@ def compute_planar_stiffness_matrix(l: Array, A: Array, Ib: Array, E: Array, G:
     S = l * jnp.diag(jnp.stack([Ib * E, 4 / 3 * A * G, A * E], axis=0))
 
     return S
+
+def gauss_quadrature(
+    N_GQ: int, 
+    a = 0.0,
+    b = 1.0
+)-> Tuple[Array, Array, int]:
+    """
+    Computes the Legendre-Gauss nodes and weights on the interval [0, 1] 
+    using Legendre-Gauss Quadrature with truncation order N_GQ.
+
+    Args:
+        N_GQ (int): order of the truncature.
+        a (float, optional): The lower bound of the interval. Default is 0.0.
+        b (float, optional): The upper bound of the interval. Default is 1.0.
+
+    Returns:
+        tuple: A tuple containing:
```

Comment:
You don't need this `tuple` here in the caption. You can directly state the contents of the tuple as the outputs of the function.